server/candlepin.spec.tmpl

```diff
@@ -73,13 +73,8 @@ Summary:        SELinux policy module supporting candlepin
 Group:          System Environment/Base
 BuildRequires:  checkpolicy
 BuildRequires:  selinux-policy-devel
-BuildRequires:  /usr/share/selinux/devel/policyhelp
 BuildRequires:  hardlink
 
-%{!?_selinux_policy_version: %global _selinux_policy_version %(sed -e 's,.*selinux-policy-\\([^/]*\\)/.*,\\1,' /usr/share/selinux/devel/policyhelp 2>/dev/null)}
-%if "%{_selinux_policy_version}" != ""
-Requires:      selinux-policy >= %{_selinux_policy_version}
```

Comment:
We need to keep this Requires line, as we still want to ensure they get an updated policy generally when they install the candlepin selinux policy.

Reply:
ACK, will do.